app.js

```diff
@@ -69,13 +69,20 @@ var app = express();
 //https://expressjs.com/en/advanced/best-practice-security.html
 //https://helmetjs.github.io/docs/
 const helmet = require('helmet');
-app.use(helmet());
+
+app.use(helmet({
+  contentSecurityPolicy: false, 
+  crossOriginEmbedderPolicy: false,
```

Comment:
Do we really need to change all these values. I think just removing `upgrade-insecure-requests` option from `contentSecurityPolicy` header should be sufficient

Reply:
Based on my understanding, the errors are related to the CORs policies and `upgrade-insecure-requests` directive. Removing `upgrade-insecure-requests` directive, rather than disabling `contentSecurityPolicy`, will work alongside with the disabled CORS policies but alone will not resolve the errors seen in load testing.